lib/active_support/cache/database_store.rb

```diff
@@ -144,10 +174,18 @@ def read_multi_entries(names, options)
         results
       end
 
+      def compress(object, compression)
```

Comment:
these should all be private

Reply:
All of those are private, since methods are listed under private directive.